src/core/Browser.js

```diff
@@ -100,7 +100,7 @@ export var pointer = !webkit && !!(window.PointerEvent || msPointer);
 // This does not necessarily mean that the browser is running in a computer with
 // a touchscreen, it only means that the browser is capable of understanding
 // touch events.
-export var touch = !window.L_NO_TOUCH && (pointer || 'ontouchstart' in window ||
+export var touch = !window.L_NO_TOUCH && (pointer || 'ontouchstart' in window || navigator.maxTouchPoints > 0 || navigator.msMaxTouchPoints > 0 ||

```

Comment:
Proposed change is not correct, as [`Browser.touch`](https://leafletjs.com/reference-1.6.0.html#browser-touch) has different meaning, and should not depend on `navigator.maxTouchPoints`.
Compare with #6978.